template.md

```diff
@@ -22,7 +22,9 @@ Across the entire IPFS GitHub organization, the following people have committed
 
 - __Contributors__
 
-Thanks, and see you next week!  If you have cool things to share for the next weekly, drop us a line in the [next weekly sprint issue](//github.com/ipfs/pm/issues/83)!
+This newsletter is also made by contributors. If you have cool things to share for the next weekly, drop a comment about it in the [next weekly sprint issue](//github.com/ipfs/pm/issues/83)! The more people mention items they want to see in the weekly there, the easier it is to make this and send it out.
```

Comment:
- maybe better: "made by **the** contributors" ? 
- A bit confused on what you are trying to say